pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_vaccination_data.py

```diff
@@ -70,31 +73,55 @@ class TestGetVaccinationData(fake_filesystem_unittest.TestCase):
     df_vacc_data = df_vacc_data.astype({'LandkreisId_Impfort': 'string', 'Altersgruppe': "string",
                    'Impfschutz': int, 'Anzahl': int})
 
+    df_vacc_data_altern = pd.DataFrame(columns=col_names_vacc_data)
+    for i in range(len(counties)):
+        vacc_data_altern = [
+            ('2020-12-27', str(counties[i]), '02-03', 2, i),
+            ('2020-12-27', str(counties[i]), '04-10', 2, i),
+            ('2020-12-27', str(counties[i]), '11-17', 2, i),
+            ('2020-12-27', str(counties[i]), '18-55', 2, i),
+            ('2020-12-27', str(counties[i]), '56+', 2, i),
+        ]
+        df_to_append = pd.DataFrame(
+            vacc_data_altern, columns=col_names_vacc_data)
+        df_vacc_data_altern = df_vacc_data_altern.append(
+            df_to_append, ignore_index=True)
+
+    df_vacc_data = df_vacc_data.astype({'LandkreisId_Impfort': 'string', 'Altersgruppe': "string",
```

Comment:
```suggestion
    df_vacc_data_altern = df_vacc_data.astype({'LandkreisId_Impfort': 'string', 'Altersgruppe': "string",
```

Reply:
I changed this manually with the commit since the second vacc_data also had to be replaced.